This patch was figured out by @shoes22 here:
https://forum.xda-developers.com/showpost.php?p=70596009&postcount=1450

Interesting situation.  LG hasn't been releasing the source code of their
/actual/ kernels...

--- kernel/msm-3.18.orig/arch/arm64/boot/dts/lge/dsi-panel-sw49407-dsc-qhd-cmd.dtsi	2016-09-17 01:59:44.000000000 -0700
+++ kernel/msm-3.18/arch/arm64/boot/dts/lge/dsi-panel-sw49407-dsc-qhd-cmd.dtsi	2017-01-17 20:57:57.914555535 -0800
@@ -40,7 +40,7 @@
 		qcom,mdss-dsi-bllp-eof-power-mode;
 		qcom,mdss-dsi-bllp-power-mode;
 		qcom,partial-update-enabled;
-		qcom,panel-roi-alignment = <1440 160 1440 160 1440 160>;
+		qcom,panel-roi-alignment = <1440 16 1440 16 1440 16>;
 		qcom,mdss-dsi-lane-0-state;
 		qcom,mdss-dsi-lane-1-state;
 		qcom,mdss-dsi-lane-2-state;
